charts/pulsar/values.yaml

```diff
@@ -278,7 +278,7 @@ zookeeper:
   replicaCount: 3
   updateStrategy:
     type: RollingUpdate
-  podManagementPolicy: OrderedReady
+  podManagementPolicy: Parallel
```

Comment:
@codelipenghui Do we need this change for #190 ? Can you remember the reason for this change? /cc @michaeljmarshall @315157973

Reply:
I think `Parallel` is the right choice for us. It'll lead to faster cluster initialization. https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#parallel-pod-management